com.microsoft.mrtk.core/Utilities/Extensions/VectorExtensions.cs

```diff
@@ -169,7 +169,7 @@ public static bool IsValidVector(this Vector3 vector)
         /// <param name="v2">The second vector.</param>
         /// <param name="distanceTolerance">The maximum distance that will cause this to return true.</param>
         /// <returns>True if the distance between the two vectors is within the tolerance, false otherwise.</returns>
-        public static bool CloseEnough(Vector3 v1, Vector3 v2, float distanceTolerance)
+        public static bool CloseEnoughTo(this Vector3 v1, Vector3 v2, float distanceTolerance = 0.001f)
         {
             return Mathf.Abs(Vector3.Distance(v1, v2)) < distanceTolerance;
```

Comment:
nit (though outside your actual changes): we shouldn't need to take the absolute value of a distance 🤪

Reply:
Fixed